sources/engine/Xenko.VirtualReality/TrackingSpace.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Xenko contributors (https://xenko.com) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
```

Comment:
New files should only be `// Copyright (c) Xenko contributors (https://xenko.com)` without Silicon Studio